Answers.md

```diff
@@ -0,0 +1,5 @@
+An adaptive website is the creation of multiple web pages to fit each screen size. A fully responsive website is the abiltiy to create just one webpage that adapts to each screen size. 
+Mobile first is creating css that starts at the mobile size and works its way up to the desktop size, where desktop first is the opposite in that it starts at the biggest screen size and works its way down to mobile. 
+When we set our default font-size to 62.5%, it allows us to understand that our default font size is 16px or 1rem.  This way when we want to change to other font sizes we know how much 2rem would be or how to make a smaller font size. 
```

Comment:
When we set our default font-size to 62.5%, we are forcing 1 rem to equal 10px, which makes converting between rem and px much easier than if 1 rem were to remain equal to 16px instead.